antlr4ide.ui/src/main/java/com/github/jknack/antlr4ide/ui/console/DefaultConsole.java

```diff
@@ -0,0 +1,77 @@
+package com.github.jknack.antlr4ide.ui.console;
+
+import com.google.common.base.Objects;
+import org.eclipse.debug.ui.IDebugUIConstants;
+import org.eclipse.debug.ui.console.ConsoleColorProvider;
+import org.eclipse.swt.graphics.Color;
+import org.eclipse.swt.widgets.Display;
+import org.eclipse.ui.console.IOConsole;
+import org.eclipse.ui.console.IOConsoleOutputStream;
+import org.apache.log4j.AppenderSkeleton;
+import org.apache.log4j.Level;
+import org.apache.log4j.spi.LoggingEvent;
+
+public class DefaultConsole extends AppenderSkeleton  {
+	
+	public DefaultConsole() {
+		super();
+		System.out.println("DefaultConsole - constructor");
```

Comment:
Ah thank you! As I already wrote, I was wondering why the ConsoleView never printed anything. Will remove them soon!